mobify/progressive-web-scaffold
# Login page

Add a login page to showcase a working form

 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/137

## Changes
- Adds a route for the login page
- Adds a LoginForm component with form validation that can be submitted

## How to test-drive this PR
- Checkout this branch of the SDK: https://github.com/mobify/progressive-web-sdk/pull/137
- In the SDK folder, run `npm link`
- Back in the scaffold folder, checkout this branch and run `npm i && npm link progressive-web-sdk`
- `npm run dev`
- Navigate to the login page and make sure the form works, and alerts the values when you submit it.

# TODO
- [x] Change import back from `progressive-web-sdk/src/components/field` to `progressive-web-sdk/dist/components/field`
